middlewares/campaigns.js

```diff
@@ -33,14 +35,32 @@ exports.registerUser = async (req, res) => {
             transaction: t
         });
 
+        // TODO: add duplicate name checking; translit gsuite_email in case of umlauts
+        const gsuite_email = req.body.first_name+'.'+req.body.last_name+'@aegee.eu'
+        const payload = {
+            'primaryEmail': gsuite_email.toLowerCase().replace(' ',''),
+            'name': {
+              'givenName':  req.body.first_name,
+              'familyName': req.body.last_name
+            },
+            'secondaryEmail': req.body.email,
+            'password': crypto.createHash('sha1').update(JSON.stringify(req.body.password)).digest('hex'),
```

Comment:
is sha-1 secure enough?

Reply:
it's how google wants it transmitted, can't really say more

Reply:
okay then, I was wondering because github was complaining, but this isn't a big deal